RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducer.cc

```diff
@@ -41,7 +41,7 @@ PixelVertexProducer::PixelVertexProducer(const edm::ParameterSet& conf)
     track_pt_min   = PVcomparerPSet.getParameter<double>("track_pt_min");    
     if (track_pt_min != ptMin_) {
       if (track_pt_min < ptMin_)
-	edm::LogWarning("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
+	edm::LogInfo("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
```

Comment:
we really need more than 5 billion MC events worth of the printout?

> On Oct 21, 2015, at 3:38 PM, Andrea Bocci <notifications@github.com> wrote:
> 
> In RecoPixelVertexing/PixelVertexFinding/src/PixelVertexProducer.cc:
> 
> > @@ -41,7 +41,7 @@ PixelVertexProducer::PixelVertexProducer(const edm::ParameterSet& conf)
> >      track_pt_min   = PVcomparerPSet.getParameter<double>("track_pt_min");    
> >      if (track_pt_min != ptMin_) {
> >        if (track_pt_min < ptMin_)
> > -	edm::LogWarning("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
> > +	edm::LogInfo("PixelVertexProducer") << "minimum track pT setting differs between PixelVertexProducer (" << ptMin_ << ") and PVcomparer (" << track_pt_min << ") [PVcomparer considers tracks w/ lower threshold than PixelVertexProducer does] !!!";
> 
> I'd rather keep the warning (it should be once per job, at configuration time), since it points out to a possible underlying problem.
> If it is a problem for the 7.6.x MC production, I would prefer to silence it via the MessageLogger configuration.
> 
> —
> Reply to this email directly or view it on GitHub.
>